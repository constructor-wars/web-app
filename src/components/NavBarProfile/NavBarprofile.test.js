import React from 'react';
import NavbBarProfile from './NavBarProfile';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import { render } from 'enzyme';

describe('navprofilebar', () => {
  it('it matches and update progress', () => {
    const mockStore = configureStore();
    const tree = renderer
      .create(
        <MemoryRouter>
          <Provider
            store={mockStore({
              GITHUB_DATA: {
                id: 10,
                displayName: 'bob',
                profileUrl: 'bob2',
                username: 'bob 3',
                photos: [{ value: 'bob4' }]
              }
            })}
          >
            <NavbBarProfile />
          </Provider>
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
