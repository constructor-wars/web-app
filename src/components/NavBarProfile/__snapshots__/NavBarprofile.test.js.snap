// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navprofilebar it matches and update progress 1`] = `
<a
  href="/profile"
  onClick={[Function]}
>
  <div
    className="navbarprofile_container"
  >
    <img
      className="navbarprofile_picture"
    />
  </div>
</a>
`;
